src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandling.tsx

```diff
@@ -0,0 +1,122 @@
+        !aktivBehandling || aktivBehandling?.status === BehandlingStatus.AVSLUTTET;
+    const førstegangsbehandlingEnabled =
+        fagsak.status !== FagsakStatus.LØPENDE && kanOppretteBehandling;
+    const revurderingEnabled = fagsak.behandlinger.length > 0 && kanOppretteBehandling;
+
+    const {
+        fjernState,
+        onBekreft,
+        settBehandlingstype,
+        behandlingstype,
+        settSubmitRessurs,
+        submitRessurs,
+    } = useOpprettBehandling(() => settVisModal(false));
+
+    const lukkOpprettBehandlingModal = () => {
+        fjernState();
+        settVisModal(false);
+    };
+
+    return (
+        <>
+            <KnappBase
+                mini={true}
+                onClick={() => {
+                    onListElementClick();
+                    settVisModal(true);
+                }}
+            >
+                Opprett behandling
+            </KnappBase>
+
+            <UIModalWrapper
+                modal={{
+                    actions: [
+                        <Flatknapp
+                            key={'avbryt'}
+                            mini={true}
+                            onClick={lukkOpprettBehandlingModal}
+                            children={'Avbryt'}
+                        />,
+                        <Knapp
+                            key={'bekreft'}
+                            type={'hoved'}
+                            mini={true}
+                            onClick={() => onBekreft(fagsak.søkerFødselsnummer)}
+                            children={'Bekreft'}
+                            spinner={submitRessurs.status === RessursStatus.HENTER}
+                            disabled={submitRessurs.status === RessursStatus.HENTER}
+                        />,
+                    ],
+                    onClose: lukkOpprettBehandlingModal,
+                    lukkKnapp: true,
+                    tittel: 'Opprett ny behandling',
+                    visModal,
+                }}
+            >
+                <SkjemaGruppe
+                    feil={
+                        submitRessurs.status === RessursStatus.FEILET
+                            ? submitRessurs.frontendFeilmelding
+                            : ''
+                    }
+                >
+                    <FamilieSelect
+                        erLesevisning={false}
+                        value={behandlingstype}
+                        name={'Behandling'}
+                        onChange={(event: React.ChangeEvent<HTMLSelectElement>): void => {
+                            settSubmitRessurs(byggTomRessurs());
+                            settBehandlingstype(
+                                event.target.value
+                                    ? (event.target.value as Behandlingstype)
+                                    : undefined
+                            );
+                        }}
+                    >
+                        <option value={''}>Velg</option>
```

Comment:
Er verdien '' eller undefined til å begynne med?

Reply:
Sant, det skal egentlig være undefind  her for verdien er undefined. Fikser!